src/App.js

```diff
@@ -274,7 +278,25 @@ export const App = ({
               <CreateAccountFreeTrial {...props} isLoggedIn={isLoggedIn} />
             )}
           />
-          <Route exact path='/assets' component={LoadableWatchlistsPage} />
+          <Route
+            exact
+            path='/assets'
+            render={() => <LoadableAssetsPage isDesktop={isDesktop} />}
+          />
+          <Route
+            exact
+            path='/screener/:nameId'
+            render={props => (
+              <LoadableScreenerPage {...props} isDesktop={isDesktop} />
```

Comment:
When user logged in and go to `/screener/new` we create his own screener, make auto-redirect to fresh list and he can save filters.
`isLoggedIn` prop here for that.
```suggestion
              <LoadableScreenerPage {...props} isDesktop={isDesktop} isLoggedIn={isLoggedIn} />
```